maggy/core/rpc.py

```diff
@@ -540,173 +538,3 @@ def _tf_callback(self, resp: dict, *_: Any) -> None:
         except KeyError:
             resp["data"] = None
         resp["type"] = "OK"
-
-
-class Client(MessageSocket):
```

Comment:
why did you have to move the client?

Reply:
yes it doesn't make so much sense actually, I'll change it back.